package.json

```diff
@@ -53,6 +53,7 @@
     "localforage": "^1.10.0",
     "lodash": "^4.17.21",
     "match-sorter": "^6.3.0",
+    "nock": "^13.2.2",
```

Comment:
Remove `nock` from this PR. We may end up using `msw` instead so let's keep the fixes for the market-service separate from the network mock library.

Reply:
Sure ! done